crates/router/src/core/payment_methods/vault.rs

```
@@ -977,9 +977,8 @@ pub async fn create_tokenize(
             )
             .await
             .map(|_| lookup_key.clone())
-            .map_err(|err| {
+            .inspect_err(|_| {
```

instead discarding the error, log there error also